packages/scheduler/src/__tests__/SchedulerSetTimeout-test.js

```diff
@@ -23,6 +23,7 @@ describe('SchedulerNoDOM', () => {
     jest.resetModules();
     jest.useFakeTimers();
     delete global.setImmediate;
+    delete global.MessageChannel;
```

Comment:
Jest v28 add MessageChannel, MessageEvent to globals. https://github.com/facebook/jest/pull/12553